### ISSUE TITLE
cant run on windows [Errno 2] No such file or directory: 'E\\dict.ru.txt'

### ISSUE THREAD

**Issue by hadaev8** [pytorch/fairseq#1363]
With this code
```
import torch

# List available models
torch.hub.list('pytorch/fairseq', 'transformer.wmt19.ru-en')

# Load a transformer trained on WMT'16 Ru - En
ru2en = torch.hub.load('pytorch/fairseq', 'transformer.wmt19.ru-en', checkpoint_file='model1.pt:model2.pt:model3.pt:model4.pt',
                       tokenizer='moses', bpe='fastbpe')
```
I got error
`[Errno 2] No such file or directory: 'E\\dict.ru.txt'`
This file actually in cash folder.
Also even if i put file on disc E, it should be E:\\dict.ru.txt

**Comment by huihuifan** [comment]
please see: https://github.com/pytorch/fairseq/issues/1370 perhaps

**Comment by myleott** [comment]
Fixed by cd521b4a7c7334c5bac08063bdf73768a7a70c9f
